README.md

```diff
@@ -88,7 +88,8 @@ repository.get('oai:www.example.com:12345')
   * [`#sets`](#fieldhandheadersets)
   * [`#response_date`](#fieldhandheaderresponse_date)
 * [`Fieldhand::NetworkError`](#fieldhandnetworkerror)
-  * [`Fieldhand::NetworkError#response`](#fieldhandnetworkerrorresponse)
+* [`Fieldhand::ResponseError`](#fieldhandresponseerror)
```

Comment:
yep that makes sense. should I also change the paragraph titles from h3 to h4 and  h4 to h5?